pkg/controller/v1alpha2/trial/trial_controller_util.go

```diff
@@ -96,29 +83,23 @@ func UpdateTrialStatusCondition(instance *trialsv1alpha2.Trial, deployedJob *uns
 	return nil
 }
 
-func UpdateTrialStatusObservation(instance *trialsv1alpha2.Trial, deployedJob *unstructured.Unstructured) error {
-
-	// read GetObservationLog call and update observation field
+func (r *ReconcileTrial) UpdateTrialStatusObservation(instance *trialsv1alpha2.Trial, deployedJob *unstructured.Unstructured) error {
 	objectiveMetricName := instance.Spec.Objective.ObjectiveMetricName
-	request := &api_pb.GetObservationLogRequest{
-		TrialName:  instance.Name,
-		MetricName: objectiveMetricName,
-	}
-	if reply, err := common.GetObservationLog(request); err != nil {
+	reply, err := r.GetTrialObservationLog(instance)
+	if err != nil {
```

Comment:
Can you add a error og here? Missed earlier

Reply:
SGTM